Source/WebCore/platform/graphics/Font.h

```diff
@@ -241,6 +247,11 @@ class Font : public RefCounted<Font>, public CanMakeSingleThreadWeakPtr<Font> {
 #endif
 #endif
 
+#if USE(SKIA)
+    sk_sp<SkTextBlob> buildTextBlob(std::span<const GlyphBufferGlyph>, std::span<const GlyphBufferAdvance>, FontSmoothingMode) const;
+    bool enableAntialiasing(FontSmoothingMode) const;
```

Comment:
We usually use prefixes like "use" / "can" / "should" for boolean conditions like this: "bool shouldEnableAntialiasing".

Reply:
I'm not sure we are very consistent, see `FontCascade::enableKerning()` for example...